test/test_browser.py

```diff
@@ -3363,8 +3363,12 @@ def test_async_mainloop(self, args):
     self.btest_exit('async_mainloop.cpp', args=args + ['-sASYNCIFY'])
 
   @requires_sound_hardware
-  def test_sdl_audio_beep_sleep(self):
-    self.btest_exit('test_sdl_audio_beep_sleep.cpp', args=['-Os', '-sASSERTIONS', '-sDISABLE_EXCEPTION_CATCHING=0', '-profiling', '-sSAFE_HEAP', '-lSDL', '-sASYNCIFY'], timeout=90)
+  @parameterized({
+    '': ([],),
+    'safeheap': (['-sSAFE_HEAP'],),
+  })
+  def test_sdl_audio_beep_sleep(self, args):
```

Comment:
Unrelated change to @paramaterize this test.. maybe I can sneak this in without a separate PR?

Reply:
sgtm. Separately this makes me wonder if a `@also_with_safe_heap` decorator could be useful...